bitwarden_license/bit-web/src/app/sm/secrets/dialog/secret-dialog.component.html

```diff
@@ -0,0 +1,43 @@
+<bit-dialog dialogSize="default">
+  <span bit-dialog-title>{{ title | i18n }}</span>
+  <span bit-dialog-content>
+    <!-- Need to get tab groups working; right now just place holder-->
+    <bit-tab-group class="tw-w-full">
+      <bit-tab-item [route]="['name-value']">{{ "nameValuePair" | i18n }}</bit-tab-item>
+      <bit-tab-item [route]="['service-accounts']">{{ "serviceAccounts" | i18n }}</bit-tab-item>
+      <bit-tab-item [route]="['projects']">{{ "projects" | i18n }}</bit-tab-item>
+    </bit-tab-group>
+
+    <!-- NameValue pair block-->
+    <form [formGroup]="form">
+      <div class="tw-flex tw-gap-5">
+        <bit-form-field class="tw-w-1/3 tw-pt-4 tw-pl-4">
+          <bit-label for="secret-name">{{ "name" | i18n }}</bit-label>
+          <input formControlName="name" bitInput />
+        </bit-form-field>
+        <bit-form-field class="tw-w-full tw-pt-4 tw-pr-24">
+          <bit-label>{{ "value" | i18n }}</bit-label>
+          <textarea bitInput rows="4" formControlName="value"></textarea>
+        </bit-form-field>
+      </div>
+      <bit-form-field class="tw-pl-4 tw-pr-24">
+        <bit-label>{{ "notes" | i18n }}</bit-label>
+        <textarea bitInput rows="4" formControlName="notes"></textarea>
+      </bit-form-field>
+    </form>
+  </span>
+  <div bit-dialog-footer class="tw-flex tw-flex-row tw-gap-2">
```

Comment:
Row is implicit with flex.

```suggestion
  <div bit-dialog-footer class="tw-flex tw-gap-2">
```

Reply:
Resolved in https://github.com/bitwarden/clients/pull/3376/commits/36d309fc40884cd9fccc2a43edaf2da34f34404b